text-utils/col.c

```diff
@@ -651,5 +682,6 @@ int main(int argc, char **argv)
 		/* missing a \n on the last line? */
 		ctl.nblank_lines = 2;
 	flush_blanks(&ctl);
+	free_line_allocations(ctl.alloc_root);
```

Comment:
Maybe we can use some #ifdef to call free_line_allocations() only to make LeakSanitizer happy :-)

Reply:
Lets see if 3252914310e7a8d76f1a3fab7371a2ff278b2419 works as expected. I added `#if defined(__SANITIZE_ADDRESS__)` around code that is performing these pointless at exit free's.